src/main/kotlin/three/consulting/epoc/dto/TaskDTO.kt

```diff
@@ -12,6 +12,7 @@ data class TaskDTO(
     val project: ProjectDTO,
     val created: LocalDateTime? = null,
     val updated: LocalDateTime? = null,
+    val billable: Boolean = true,
```

Comment:
You can add the @field:NotNull annotation to make sure the openapi defintions work, like done here: https://github.com/three-consulting/epoc-backend/pull/78